app/assets/stylesheets/navigation.scss

```diff
@@ -29,12 +32,35 @@
 }
 #search_bar {
   width: 115px;
+  border-top: 0px;
+  border-right: 0px;
+  border-left: 0px;
+  box-shadow: none;
+  border-color: #ffffff;
+}
+
+#search_bar:hover {
+  border-color: $neutral;
+}
+
+#search_bar:focus {
+  border-color: $neutral;
 }
 
 #search_button {
   width: 95px;
 }
 
+#search_button > button {
```

Comment:
Avoid using id selectors<br>Merge rule `#search_button > button` with rule on line 50<br>Selector `search_button` should be written in lowercase with hyphens